### DISCUSSION TITLE
SDK en varios procesos.

### DISCUSSION THREAD

**Discussion by palacejohn** [AndresRamos/Contpaqi#91] [Q&A]
Hola! buen día, en el Wiki mencionas que se debe iniciar el SDK y luego abrir una empresa y que solo se puede abrir una empresa a la vez, en tus cursos comentas que se puede abrir el SDK una vez por proceso.

Bien, mi duda, ¿es posible iniciar varias veces el SDK? en diferentes procesos, como una vez en una consoleApp y abrir nuevamente en una WPF abriendo diferentes empresas cada uno, ¿es posible eso que trabajen simultaneamente o el SDK no lo permite?

**Comment by AndresRamos** [comment]
Creo que si es posible hacerlo, pero no lo recomendaría. Tendria que hacer pruebas. Un proceso de consola en el servidor y WPF en terminales al mismo tiempo si funciona. Pero tu quieres saber en la misma maquina verdad? Creo que pudieras hacer pruebas corriendo los ejemplos simultáneamente.

**Reply by palacejohn** [reply]
Interesante! hare pruebas y te cuento. Me va ayudar mucho editar diferentes empresas a la vez si funciona, pues en un contpaqi con 10 empresas será muy lento crear facturas; empresa por empresa factura por factura.
